regexes/device/mobiles.yml

```diff
@@ -29875,7 +29875,7 @@ PocketBook:
       model: 'A10'
     - regex: 'SURFpad2(?:[);/ ]|$)'
       model: 'SURFpad 2'
-    - regex: 'PocketBook SURFpad 3 \(([\d]+).([\d]+)["]?\)(?:[);/ ]|$)'
+    - regex: 'PocketBook SURFpad 3 \(([\d]+)\.([\d]+)["]?\)(?:[);/ ]|$)'
```

Comment:
```suggestion
    - regex: 'PocketBook SURFpad 3 \(([\d]+).([\d]+)["]?\)(?:[);/ ]|$)'
```

Reply:
Oh my god, this is matching `PocketBook SURFpad 3 (10,1)`, they're using a comma for the version number! I'll change this to a comma.

Reply:
be1ae923